src/main.cc

```diff
@@ -85,101 +92,157 @@ Global Options:
 
 For more documentation, visit https://github.com/sourcemeta/jsonschema
 )EOF"};
+// Initialize color usage
+bool use_colors = true;
```

Comment:
We shouldn't have a boolean like this. Instead, each command should check the command line arguments + the TTY check.

Reply:
Alright ,in our code i was just going to check ones when the problem starts.Thankyou for guidance i will make sure it checks each time when command run.